atd-vzd/migrations/migration_fatalities_2023_04_27--1526.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE fatalities ADD COLUMN updated_by text;
```

Comment:
keeps track of what email edited the victim name

Reply:
probably out of scope, but if the user email is in the user's session token, it would be very cool to set this value via [Hasura permission rule](https://hasura.io/docs/latest/auth/authorization/permissions/column-presets/).

Reply:
ohh cool, TIL! this is also a reminder to fix how we are granting users that have "Admin" access in the VZE actual admin privileges in hasura. bc from what i can tell thru the console you cant set column presets for the admin role